frontend/src/components/templates/form.field.text-input.js

```diff
@@ -45,6 +47,68 @@ const _wbTextInputFieldTemplate = (props) => {
     `.trim();
 };
 
+/**
+ * Function that returns Waterboard attachment upload input field template string
+ * @param props
+ * @returns {string}
+ * @private
+ */
+const _wbAttachmentUploadInputFieldTemplate = (props) => {
+
+    const {
+        key,
+        label,
+        required,
+        multiple,
+        type='file',
+        value='[]',
+        className='form-group',
+        labelClassName='control-label',
+        inputClassName='form-control',
+        inputAttributes=[]
+    } = props;
+
+    let fieldAttrs = buildAttributeString(inputAttributes);
+
+    let parsedValue = [];
+    if (value.length > 0) {
+        parsedValue = JSON.parse(value);
+    }
+
+    const tmplAttachments = parsedValue.map(
+        item => `
+          <li>
+            <a target="_blank" href="/api/v1/attachments/${item.attachment_uuid}/">${item.filename}</a>
```

Comment:
Href param should be set from props argument.
The base path `/api/v1/attachments/` should be part of props arguments.